docs/pages/release_notes.rst

```diff
@@ -12,6 +12,7 @@ Coconut update (v5.1.0)
 Release date: **DRAFT**
 
 - Default value of strictmode changed to true
+- Internal storage of VDR has changed. A migration will run at startup. If the node is stopped during this process, DID Documents will have to be reprocessed manually (restore functionality)
```

Comment:
If `Migrate()` calls are not atomic (1 db TX), we probably shouldn't allow the node to be stopped during migration (but it probably won't, since signal handlers are registered after `Start()` has finished, I think).

Reply:
`kill -9` is unstoppable

Reply:
That would be a different issue, this PR only fixes a required restart after updating.

Reply:
I'm a bit weary about atomic operations and Redis:

```
// Pay attention, that Pipeline is not a transaction, so you can get unexpected
// results in case of big pipelines and small read/write timeouts.
// Redis client has retransmission logic in case of timeouts, pipeline
// can be retransmitted and commands can be executed more then once.
// To avoid this: it is good idea to use reasonable bigger read/write timeouts
// depends of your batch size and/or use TxPipeline.
```